### DISCUSSION TITLE
Valent stays open after closing the main window, doesn't have a tasktray icon

### DISCUSSION THREAD

**Discussion by colemickens** [andyholmes/valent#430] [Feature Request]
In lieu of https://github.com/andyholmes/valent/discussions/403, I don't think Valent should keep running in the background, or I think there should be an option to disable it.

**Comment by andyholmes** [comment]
There's a `Quit` option in the main menu, but the expected behaviour of Valent is to run as a service. It's actually just an unfortunate necessity that there's a UI at all, to be honest :)

For Flatpak users the `Background Apps` portal can be used to restrict Valent and the user will be prompted before it is allowed to start at login. For other sources, a XDG Autostart file is installed by default so the application will usually start by itself anyways.

I guess this is something distributions could make a decision about, whether they want it to auto-start or not.
